markdown/pxf/HivePXF.html.md.erb

```diff
@@ -565,6 +577,44 @@ In the following example, you will create a Hive table stored in ORC format and
     Time: 425.416 ms
     ```
 
+### <a id="using-hiveorc-profile"></a>Example: Using the HiveVectorizedORC Profile
+
+In the following example, you will use the `HiveVectorizedORC` profile to query the `sales_info_ORC` Hive table you created in the previous example.
+
+**Note**: The `HiveVectorizedORC` profile does not support the timestamp data type and complex types.
```

Comment:
Just to avoid any potential confusion, let's change this to "**or** complext types."